sql/core/src/main/scala/org/apache/spark/sql/DataFrameStatFunctions.scala

```diff
@@ -72,13 +73,38 @@ final class DataFrameStatFunctions private[sql](df: DataFrame) {
   }
 
   /**
+   * Calculates the approximate quantiles of numerical columns of a DataFrame.
+   *
+   * Note that rows containing any null values will be removed before calculation.
```

Comment:
Let's actually change both this line and the one for the above `approxQuantile` method to say `Note that rows containing any null or NaN values will be removed ...`